application/frontend/controllers/UserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace frontend\controllers;
+
+use common\models\User;
+use frontend\components\BaseRestController;
+use Yii;
+use yii\web\NotFoundHttpException;
+
+class UserController extends BaseRestController
+{
+    public function actionCreate(): User
+    {
+        $existingUser = User::findOne([
+            'employee_id' => Yii::$app->request->getBodyParam('employee_id')
+        ]);
+
+        $user = $existingUser ?? new User();
+
+        $user->attributes = Yii::$app->request->getBodyParams();
```

Comment:
I'm sorry to bring this up again (and maybe it's just a mindset shift I need to work through), but I still cringe at using massive assignment to collect data from the request and save it to the database. It feels like we're completely tossing out the opportunity to effectively do a whitelist for what properties we are willing to accept from requests. Does that seem reasonable, or am I just obsessing over something that doesn't matter?